Pod/Classes/AccessibilityHelpers/AccessibilityHelpers.swift

```diff
@@ -0,0 +1,84 @@
+//
+//  AccessibilityHelpers.swift
+//  Pods
+//
+//  Created by Michael Skiba on 11/17/16.
+//
+//
+
+import UIKit
+
+public typealias AccessibilityAnnounceCompletion = (_ anncouncedString: String?, _ success: Bool?) -> ()
```

Comment:
#nth I just replaced this `Bool?` with a `Bool` on my project, and used `[get the bool from the dictionary] ?? false` where it's called. Optional Booleans are annoying, and they make the implementation of the completion into this mess:

```swift
if let success = success, success {
    // what even
}
```